config/dependency-injection/custom-loader.php

```diff
@@ -187,4 +191,15 @@ private function findClasses( $namespace, $pattern, $exclude ) {
 
 		return $classes;
 	}
+
+	/**
+	 * Normalize all slashes in a file path to forward slashes.
```

Comment:
Very minor: method descriptions should start with a third-person verb. Will adjust during CR.